internal/pkg/manifest/validate.go

```diff
@@ -616,6 +619,21 @@ func (c NetworkLoadBalancerConfiguration) Validate() error {
 	return nil
 }
 
+func validateNLBPort(port *string) error {
+	_, protocol, err := ParsePortMapping(port)
+	if err != nil {
+		return err
+	}
+	if protocol == nil {
+		return nil
+	}
+	protocolVal := aws.StringValue(protocol)
+	if strings.ToLower(protocolVal) != "tcp" && strings.ToLower(protocolVal) != "udp" && strings.ToLower(protocolVal) != "tls" {
```

Comment:
Could we make the available options for the port protocol to be enums? Also is `tcp_udp` a valid option?